Add layer quality checklist to contributor docs (#716)
nsidc/qgreenland: doc/contributor/how-to/contribute-layers.md

  * In GeoPackage (`.gpkg`) format.
  * Uses the `label` attribute name for pre-calculated labels when using
    generic styles with labels, for example `labeled_point.qml`

## Layer quality checklist

### All layers

- Abstract populated
- Citation populated
- Layer descriptions populated
- Layer description excludes data provider
- Layers'/groups' title follows date standard
- Layers'/groups' title follows capitalization standard

> - Layers'/groups' title excludes data provider (see notes for exceptions)

Reviewer: Can we link to the mentioned notes?

Author: I'm not sure. I'm going to see if I can find them in Jira? Maybe they're in the repo someplace, too.

Author: FOUND IT! https://docs.google.com/spreadsheets/d/13qzH-HGaWz0wqKbpG_9yoNpWy7K8UagSUbdQ-XS12mU/edit#gid=0

Author: `8a2831b7` populates info from the original spreadsheet notes

- No fields are populated with "TBD" or "TODO"
- All layers have style
- Any known issues with data quality are listed in "Description"

> - Datastore's data is in final state

Reviewer: What does this mean? That the data have been processed?

Author: I wish I knew... maybe we should remove this one.

Author: e63c8c4a

- Layers not in unneccessary group
- PO approves citation

> - PO approves description

Reviewer: Maybe write out what `PO` means? Or use `Stakeholders` instead of `PO`? Community members probably would not understand what this means.

Author: 9ba9d6f7

### Raster layers

> - Interpolation algorithm OK? (almost everything is currently bilinear, see notes

Reviewer: ```suggestion
- When reprojecting (e.g., via `gdalwarp`) or adding overviews (e.g., via `gdaladdo`), ensure that the interpolation algorithm is appropriate. (almost everything is currently bilinear, see notes
```

Provide context for when/where to check for interpolation algorithm

Author: 59334b71

  for exceptions)
- Grid resolution in layer title or group (incl. units)?

> - Target resolution specified (see notes for exceptions)

Reviewer: ```suggestion
- When reprojecting (e.g., via `gdalwarp`), ensure that the target resolution is specified (see notes for exceptions)
```

Provide context for where target resolution is needed

### Vector layers

> - Polygon labels inside polygons

Reviewer: I was going to suggest adding a link to the `style-guide`, but that's lacking this detail. Might be worth adding this there?

Author: Good catch

Author: 9989c7d9

> - All attributes have reasonable types

Reviewer: ```suggestion
- All attributes have reasonable types (e.g,. a `population` attribute should be an integer type, not string)
```
